packages/element-templates-json-schema-shared/src/defs/base.json

```
@@ -6,25 +6,30 @@
     "properties"
   ],
   "properties": {
+    "$schema": {
+      "$id": "#/$schema",
+      "type": "string",
+      "description": "URL pointing to the schema that defines the structure of the element template .json file."
+    },
     "name": {
       "$id": "#/name",
       "type": "string",
-      "description": "The name of the element template."
+      "description": "Template name. Shown in template selection modal and properties panel. Required."
```

What is template selection modal? I'd keep it agnostic of such implementation details, i.e.


```suggestion
      "description": "Template name. Shown during template selection and editing of a templated element . Required."
```


We have to consider uses of element templates outside of our applications, where "selection modal" does not exist.